extensions/wikia/CommunityPage/CommunityPageSpecialHooks.class.php

```diff
@@ -71,6 +72,12 @@ public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 			\Wikia::addAssetsToOutput( 'community_page_benefits_js' );
 			\Wikia::addAssetsToOutput( 'community_page_benefits_scss' );
 		}
+
+		if ( !$out->getUser()->isAnon() ) {
```

Comment:
I think you should check is `community-page-first-time` cookie set here to avoid unnecessary assets load